contrib/statesync.bash

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# microtick and bitcanna contributed significantly here.
+# Pebbledb state sync script.
+# invoke like: bash contrib/statesync.bash
+
+## USAGE RUNDOWN
+# Not for use on live nodes
+# For use when testing.
+
+set -uxe
+
+# Set Golang environment variables.
+# ! Adapt as required, depending on your system configuration
+#export GOPATH=~/go
+#export PATH=$PATH:~/go/bin
+
+# Install with pebbledb (uncomment for incredible performance)
+go mod edit -replace github.com/tendermint/tm-db=github.com/baabeetaa/tm-db@pebble
+go mod tidy
```

Comment:
I am not sure that we need to modify sources in this script even in this way. It is good to use on local installation, but there is risk that this changes will fall into sources. And it makes neutrond client incompatible with available snapshots. This requires additional discussion and decision.